lib/src/Uncertainty/Algorithm/WeightedExperiments/LHSResult.cxx

```diff
@@ -92,6 +93,8 @@ UnsignedInteger LHSResult::getNumberOfRestarts() const
 /** Attributes for getting elements of result */
 Sample LHSResult::getOptimalDesign() const
 {
+  if (!optimalIndex_)
+    throw InvalidArgumentException(HERE) << "No design stored!";
```

Comment:
what if optimalIndex_ is zero ? isnt it better to check the size of the sample ?

Reply:
nevermind

Reply:
:-)